data/korath/korath ships.txt

```diff
@@ -383,89 +383,845 @@ ship "'olofez"
+	explode "huge explosion" 50
+	"final explode" "final explosion large"
+	description "The largest of Korath production vessels are derived from a common and ancient chassis. This deceptively spindly ship bristles with weapon emplacements for heavy raiding or defending other vessels."  
+
+
+ship "Kas'lor Ik 582"
+	sprite "ship/world-ship c"
+	thumbnail "thumbnail/world-ship c"
+	attributes
+		category "Heavy Freighter"
+		cost 27690000
+		shields 39750
+		hull 26100
+		"required crew" 314
+		"bunks" 551
+		"mass" 1899
+		"drag" 17
+		"heat dissipation" .35
+		"solar collection" 3.8
+		"fuel capacity" 32700
+		"cargo space" 734
+		"outfit space" 839
+		"weapon capacity" 321
+		"engine capacity" 241
+		weapon
+			"blast radius" 400
+			"shield damage" 8000
+			"hull damage" 4000
+			"hit force" 12000
+	outfits
+		"Langrage Hyper-Heaver" 2
 		"Banisher Grav-Turret" 3
 		"Warder Anti-Missile" 3
 		
 		"Triple Plasma Core"
 		"Systems Core (Large)"
 		"Large Heat Shunt" 2
 		"Fuel Processor"
-		"Thermal Repeater Rifle" 150
-		"Microbot Defense Station" 19
+		"Brig"
+		"Thermal Repeater Rifle" 272
+		"Microbot Defense Station" 4
 		
 		"Thruster (Planetary Class)"
 		"Steering (Planetary Class)"
 		"Hyperdrive"
 	
+	engine -37 333 .8
+	engine 0 333 .8
+	engine 37 333 .8
+	"reverse engine" -19 -303 0.5
+		over
+	"reverse engine" 0 -303 0.5
+		over
+	"reverse engine" 19 -303 0.5
+		over
+	fighter -70 51 back
+	fighter 0 51 back
+	fighter 70 51 back
+	turret -52.5 -80.5 "Banisher Grav-Turret"
+	turret 52.5 -80.5 "Warder Anti-Missile"
+	turret -57 -50.5 "Warder Anti-Missile"
+	turret 57.5 -50.5 "Banisher Grav-Turret"
+	turret -56.5 55 "Langrage Hyper-Heaver"
+	turret 56.5 55 "Langrage Hyper-Heaver"
+	turret -80 283 "Banisher Grav-Turret"
+	turret 80 283 "Warder Anti-Missile"
+	explode "tiny explosion" 30
+	explode "small explosion" 45
+	explode "medium explosion" 50
+	explode "large explosion" 40
+	explode "huge explosion" 50
+	"final explode" "final explosion large"
+	description "The largest of Korath production vessels are derived from a common and ancient chassis. This relatively streamlined ship has been adapted to collect, refine, and transport hyperspace fuel."
+
+
+ship "Lor'nag Ik 590"
+	sprite "ship/world-ship d"
+	thumbnail "thumbnail/world-ship d"
+	attributes
+		category "Space Liner"
+		cost 29640000
+		shields 47000
+		hull 34800
+		"required crew" 538
+		"bunks" 1492
+		"mass" 1900
+		"drag" 18
+		"heat dissipation" .37
+		"solar collection" 5.6
+		"fuel capacity" 1000
+		"cargo space" 264
+		"outfit space" 856
+		"weapon capacity" 321
+		"engine capacity" 241
+		weapon
+			"blast radius" 400
+			"shield damage" 8000
+			"hull damage" 4000
+			"hit force" 12000
+	outfits
+		"Shunt-Strike Turret" 2
+		"Banisher Grav-Turret" 2
+		"Warder Anti-Missile" 2
+		"Fire-Lance" 2
+		
+		"Triple Plasma Core"
+		"Systems Core (Large)"
+		"Large Heat Shunt" 2
+		"Fuel Processor"
+		"Brig"
+		"Microbot Defense Station" 15
+		"Thermal Repeater Rifle" 272
```

Comment:
```suggestion
		"Thermal Repeater Rifle" 1492
```

Given that, when we run into these, we don't know if they're on transport duty or just on their too or from a raid, or anything; I'm inclined to just have them carry a full complement as if they were on raiding duty. That being said, given the prevalence of warfare in the Korath territories; it would be believable to me that they would have a rifle for every person on board a military transport even if it *was* just for transport duty. Never know when they might need to board someone (or fend off boarders, for that matter)..

Reply:
I think it would be better to set up two variants - one as is, with 272 rifles and 15 microbot stations as it is here, for civilian use, and a second with 1492 rifles and perhaps not using as many microbots if needed for combat outfits, for troop transport use.

Reply:
Sure, that could work for civilians. In general, though, there's really no way to tell if it's just transporting people, or soldiers, or going to a raid, or whatever. (and the way our boarding values are setup, all Korath are equally tough. (Just like all Quarg are equally good at combat. And all arach, kimek, etc.) So from a practical gameplay perspective, there's no difference between military and civilians as far as boarding combat goes.

Reply:
On the contrary, we *do* know.  This ship appears in at least two fleets already, each with different roles and behaviours.  And then there's mission appearances which can pick their own variants.

Reply:
We do though.  There are three variants of the Lor'nag Ik 590: the base, a grab-strike variant, and an explicit troop transport version not yet used but set aside for mission use.

I've gone over these now -- setting up variants is one of those tasks I tend to delegate -- and made some edits so the usage is more consistent.
 - Base 590s with minimal, defensive loadout only appear in Civilian fleets: 272 rifles, 17 microbots
 - Military 590s with Grab-Strikes only appear in hostile home fleets: 493 rifles, 15 microbots
 - Troop 590s with Shunt-Strike do not appear yet, but should take the place of a couple ground assault dredgers in missions. 1492 rifles, 8 mircobots